doc/requirements.txt

```diff
@@ -2,7 +2,8 @@ sphinx-gallery
 sphinx-bootstrap_theme
 numpydoc
 mne
-scikit-learn
 seaborn
-pandas
+scikit-learn
 joblib
+pandas
+qiskit
```

Comment:
Note: cvxpy is not included here, due to wheel failing when building numpy 16 source.